snuba/snuba_migrations/events_analytics_platform/0022_uptime_monitors_init_v2.py

```diff
@@ -0,0 +1,75 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+from snuba.utils.schemas import DateTime64
+
+storage_set = StorageSetKey.EVENTS_ANALYTICS_PLATFORM
+table_prefix = "uptime_monitor_checks_v2"
+local_table_name = f"{table_prefix}_local"
+dist_table_name = f"{table_prefix}_dist"
+
+columns: List[Column[Modifiers]] = [
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("environment", String(Modifiers(nullable=True, low_cardinality=True))),
+    Column("uptime_subscription_id", UUID()),
+    Column("uptime_check_id", UUID()),
+    Column("scheduled_check_time", DateTime64(3)),  # millisecond precision
+    Column("timestamp", DateTime64(3)),  # millisecond precision
```

Comment:
Is it really useful to have 2 timestamps that could be slightly different and store them with millisecond precision?

Reply:
One is when it was scheduled to happen, the other is when actually happened. I suppose it wouldn't affect users much if it was at second precision

Reply:
i'll update the `scheduled_check_time` to be second precision, and keep timestamp as is if that makes sense to you both, as it may end up being used in the UI / our customers could find ms precision useful if they're correlating the failed check with other things.

Reply:
Yeah, `scheduled_check_time` will never have ms anyway, so that makes sense